vizro-core/examples/visual-vocabulary/pages/magnitude.py

```diff
@@ -3,7 +3,7 @@
 import vizro.models as vm
 import vizro.plotly.express as px
 
-from pages._pages_utils import PAGE_GRID, gapminder, iris, make_code_clipboard_from_py_file, tips
+from pages._pages_utils import PAGE_GRID, gapminder, iris, make_code_clipboard_from_py_file, tips, wind
 
```

Comment:
```suggestion
from custom_charts import radar
```

Reply:
For this to work, you need to add the below code to the `custom_charts.py` - please fill out the docstrings :) 

```
@capture("graph")
def radar(data_frame, **kwargs):
    """
    Add docstrings here (see butterfly example)
    
    """
    fig = px.line_polar(data_frame, **kwargs)
    fig.update_traces(fill="toself")
    return fig
```

Reply:
Done